app-config.yaml

```diff
@@ -247,3 +247,16 @@ costInsights:
       name: Your Company's Daily Cost
     DAU:
       name: Cost Per DAU
+homepageTimer:
```

Comment:
Maybe these could be expressed as an unnumbered list instead, making it possible for people to decide how many time zones they want to show?

Reply:
+1, can use `config.getConfigArray` for this. Think it also makes sense to layer this a bit more and use `clocks` instead of `timer`. Something like this:

```yaml
homepage:
  clocks:
  - label: UTC
    timezone: UTC
... etc
```

Reply:
Good suggestion, updated to use the proposed configArray to allow any number of clocks.